tests/features/chart_source_local.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Chart local source verification
+    Partners or redhat or community can verify their charts by running the
+    chart verifier against a local chart source
```

Comment:
```suggestion
    chart verifier against a local chart source.
```

Official doc added period at the end of feature description: https://github.com/pytest-dev/pytest-bdd#example